src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -108,6 +109,23 @@ export const getModuleDetailsUma = async (
       ]
     );
     timestamp = await moduleContract.proposalHashes(proposalHash);
+  } else {
```

Comment:
Unrelated to the goal of this PR, but I think we should go back and re-examine the logic flow of this method. I think the logic/branching may be a bit more convoluted than it needs to be, and there may be error states that aren't handled gracefully (especially related to the transactions === undefined case),